search-filter/results.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * Search & Filter Pro
+ *
+ * Sample Results Template
+ *
+ * @package   Search_Filter
+ * @author    Ross Morsali
+ * @link      https://searchandfilter.com
+ * @copyright 2018 Search & Filter
+ *
+ * Note: these templates are not full page templates, rather
+ * just an encaspulation of the your results loop which should
+ * be inserted in to other pages by using a shortcode - think
+ * of it as a template part
+ *
+ * This template is an absolute base example showing you what
+ * you can do, for more customisation see the WordPress docs
+ * and using template tags -
+ *
+ * http://codex.wordpress.org/Template_Tags
+ *
+ */
+
+// If this file is called directly, abort.
+if ( ! defined('ABSPATH' ) ) {
+	exit;
+}
+
+$template_directory = get_template_directory();
+$template_url = get_template_directory_uri();
+
+global $wpdb;
+
+$query1 = <<<SQL
+select post_title from {$wpdb->prefix}posts
+where post_type='audio_post'
+and post_status='publish'
+SQL;
+
+$audio = $wpdb->get_results($query1, ARRAY_N);
+
+if ( $query->have_posts() )
+{
+	?>
+
+    <style>
+        button.show-more-button {
+            background-color: #F36D3A !important;
+            box-shadow: 0px 3px 6px #00000029 !important;
+            opacity: 80%;
+            color: white !important;
+            font-size: 20px;
+            height: 70px;
+            font-family: Roboto;
+        }
+
+        button.show-more-button.show-more-button-contracted::after {
+            content: '▼';
+            padding-left: 10px;
+        }
+
+        button.show-more-button.show-more-button-expanded::after {
+            content: '▲';
+            padding-left: 10px;
+        }
+
+        li.sf-field-search > label {
+            display: inline-flex !important;
+            align-items: center !important;
+            justify-content: space-between !important;
+        }
+
+
+        .sf-field-tag ul {
+            height: unset !important;
+        }
+
+        .share-buttons {
+            justify-content: end;
+        }
+
+        .share-buttons-loop img {
+            filter: invert();
+            height: 25px !important;
+        }
+
+        .share-buttons-loop a::after {
+            content: none !important;
+        }
+    </style>
```

Comment:
It would make things a lot more maintainable if this is actually a css file being enqueued.

Reply:
@comzeradd changes made, css moved to parent theme